portfolio.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="portfolio.css"><script src="https://kit.fontawesome.com/0a62dabd9f.js" crossorigin="anonymous"></script>
+
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    
+    <header>
+        <nav>
+            <ul class="navbar">
+                <li ><a href="#"class="black">Portfolio</a></li>
+                <li ><a href="#" class="black">About</a></li>
+                <li ><a href="#" class="black">Contact</a></li>
+            </ul>
+        </nav>
```

Comment:
As the [last code reviewer suggested]([url](https://github.com/bettcleos/Mobile-portfolio/pull/1#pullrequestreview-1126505219)) the objective of this project is to create the mobile version of your project. Kindly make sure that your project sticks as much as possible to the Figma template provided. These elements are for the desktop version of your project, not the mobile version. :eyes :eyes
Your design:
![image](https://user-images.githubusercontent.com/98363075/193269368-24812183-58cc-4399-bdce-816937f1e9f7.png)

Figma design:
![image](https://user-images.githubusercontent.com/98363075/193269626-8a364529-5573-41d7-9aa4-b5d5f38690c8.png)